A87-mtls-spiffe-support.md

```diff
@@ -0,0 +1,266 @@
+A87: mTLS SPIFFE Support
+----
+* Author(s): @markdroth, @erm-g, @gtcooke94
+* Approver: @ejona86, @dfawley
+* Status: {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2025-03-17
+* Discussion at: https://groups.google.com/g/grpc-io/c/55oIW6GNabs
+
+## Abstract
+
+We will add support for SPIFFE certificate verification in mTLS.  This
+will be supported both with TlsCredentials and via xDS.
+
+## Background
+
+[SPIFFE] is a standardized way of encoding workload identity in mTLS
+certificates.  Specifically, we want to support using the [SPIFFE bundle
+format] in place of a single CA certificate for peer certificate
+verification.  The bundle map format is defined in
+https://github.com/spiffe/spiffe/pull/304.
```

Comment:
This is a PR, and the spec is already referenced from the previous sentence.  Delete this?  Or change to "is defined in" to "was added by" - but it seems irrelevant to me.